app-libs/stf/src/challenge_code.rs

```diff
@@ -0,0 +1,95 @@
+/*
+	Copyright 2022 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use rand::Rng;
```

Comment:
I don't think we have defined `rand` crate, if I put `pub mod challenge_code` into lib.rs and try to run the test, I'll get compilation error

Reply:
strange. I don't get compilation error. *rand* crate should be depended by others. Double check it.

Reply:
I tried with **rand** dependency, but didn't save the toml. It still pass. 
Fixed.